### PR TITLE
Clean up SyliusResourceBundle configuration and add routing generation docs

### PR DESCRIPTION
**Pull request by pjedrzejewski**
Q 	|A
--------|--------
Doc fix?| 	yes
New Docs?| 	yes
Fixed tickets ?| -

### CODE REVIEW

File: bundles/SyliusResourceBundle/configuration.rst [hunk 1]
```diff
@@ -1,294 +1,80 @@
...
-In your ``app/config.yml`` (or in an imported configuration file), you need to define which resources you want to use :
+In your ``app/config/config.yml`` add:
 
 .. code-block:: yaml
 
     sylius_resource:
         resources:
-            my_app.entity_name:
+            app.book:
                 driver: doctrine/orm
```
Comment (michalmarcinkowski) [inline bundles/SyliusResourceBundle/configuration.rst]: Isn't it default driver?

**Comment by michalmarcinkowski** [comment]
We should have a configuration reference with all possible options.

File: bundles/SyliusResourceBundle/configuration.rst (hunk 1 above)
Comment (pjedrzejewski) [inline bundles/SyliusResourceBundle/configuration.rst]: Yeah, I guess we can skip that and mention that somewhere else, when we add mongodb docs etc.

Comment (michalmarcinkowski) [inline bundles/SyliusResourceBundle/configuration.rst]: :+1:

**Comment by pjedrzejewski** [comment]
@michalmarcinkowski Updated.

**Comment by michalmarcinkowski** [comment]
@pjedrzejewski how about route configuration (https://github.com/Sylius/Sylius-Docs/blob/90de7e752d7ea7fd34be4d408af1c064c0d401c1/bundles/SyliusResourceBundle/summary.rst)?

**Comment by michalmarcinkowski** [comment]
Good job! :+1:

**Pull request merged**